src/Parser/Core/Modules/Items/BFA/FirstMatesSpyglass.js

```diff
@@ -38,19 +52,21 @@ class FirstMatesSpyglass extends Analyzer {
       return;
     }
     if (event.hitType !== HIT_TYPES.CRIT) {
-      return;
+      this.timesHit += 1;
```

Comment:
The way you're calculating the crit percentage will lead to incorrect numbers; it will mark spells as crit when it didn't crit, and count crits as 2 hits and 1 crit.